src/lib/y2storage/clients/inst_prepdisk.rb

```diff
@@ -41,6 +41,11 @@ class InstPrepdisk
       def run
         return :auto if Mode.update
 
+        storage = manager.storage
+
+        storage.probed.save(Directory.logdir + "/probed-devicegraph.xml")
+        storage.staging.save(Directory.logdir + "/staging-devicegraph.xml")
+
```

Comment:
In fact, if we are going to write it to a separate file (not to y2log) with a fixed name, it's better to do it in the client instead of doing it in `StorageManager`. So it does not get overwritten with subsequent calls to `commit` (for example, running the expert partitioner in the installed system).